### PR TITLE
Initial patch - linear time MMD example

### PR DESCRIPTION
**Pull request by festusojo123**

### CODE REVIEW

File: examples/meta/src/regression/linear_time_mmd.sg.in [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#![train_test_mode]
+
+#![extract_result]
+#![extract_result]
+
+#![convex_combination]
+#![convex_combination]
+
+#![test]
+#![test]
```
Comment (gf712) [inline examples/meta/src/regression/linear_time_mmd.sg.in:22]: I am guessing you are still going to add the code here?

**Comment by festusojo123** [comment]
Yes, I am! Do you have any resources for converting pseudocode/code from other languages to the sg format?

File: examples/meta/src/regression/linear_time_mmd.sg.in (hunk 1 above)
Comment (festusojo123) [reply examples/meta/src/regression/linear_time_mmd.sg.in:22]: Yes, I am! Do you have any resources for converting pseudocode/code from other languages to the sg format?

**Comment by karlnapf** [comment]
Closing this as

* there is zero new content, copy pasted from the linear time mmd statistical test example
* why is it placed in "regression"? 
* no code in the example file

Please don't send such PRs, they spam our mailboxes and take time that could be spent helping people and giving feedback for actual PRs

**Pull request closed without merging**